Error when installing the jlab extension

I get a ModuleNotFoundError when installing the jlab extension
```
ModuleNotFoundError: Module not found: Error: Can't resolve '../styles/plugin.css' in '/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/lib'
    at factory.create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/webpack/lib/Compilation.js:535:10)
    at factory (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/neo-async/async.js:2830:7
    at /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/neo-async/async.js:6877:13
    at normalResolver.resolve (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at resolver.doResolve (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
    at resolver.doResolve (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn42 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
    at resolver.doResolve (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn1 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at hook.callAsync (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at fs.stat (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DirectoryExistsPlugin.js:22:13)
    at process.nextTick (/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:73:15)
    at processTicksAndRejections (internal/process/task_queues.js:79:9)
resolve '../styles/plugin.css' in '/home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/lib'
  using description file: /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/package.json (relative path: ./lib)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/package.json (relative path: ./styles/plugin.css)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css.wasm doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css.json doesn't exist
      as directory
        /home/martin/miniconda3/envs/geopython/share/jupyter/lab/staging/node_modules/jupyter-scales/styles/plugin.css doesn't exist
error Command failed with exit code 1.
```

--- a/js/package.json
+++ b/js/package.json
@@ -1,4 +1,4 @@
-{
+"{
   "name": "jupyter-scales",
   "version": "3.0.0",
   "description": "A widget library for scales",
@@ -11,7 +11,8 @@
     "dist/*.js",
     "dist/*.js.map",
     "dist/*.d.ts",
-    "src/**/*"
+    "src/**/*",
+    "styles/**/*"
   ],
   "scripts": {
     "clean:lib": "rimraf lib && rimraf lab",

--- a/js/package.json
+++ b/js/package.json
@@ -1,4 +1,4 @@
-"{
+{
   "name": "jupyter-scales",
   "version": "3.0.0",
   "description": "A widget library for scales",

